doc/contributor-how-to/how-to-dev-plugin.md

```diff
@@ -0,0 +1,64 @@
+# How to install QGreenland Custom as a developer
+
+QGreenland Custom is a QGIS plugin for downloading a custom set of data, including data which 
+is not part of the QGreenland Core zip package. This how-to guide walks through how to install the plugin as a developer.
+
+First, clone the QGreenland Custom repo from Faunalia:
+
+```
+git clone https://github.com/faunalia/qgreenland-plugin.git
+```
+
+Use the QGIS GUI to locate the QGreenland Custom directory by going to
+Settings -> User Profiles -> Open Active Profile Folder.
+Then, navigate to the /python/plugins directory, PWD and copy the full filpath. 
+
+Next symlink the plugin repo to the plugin directory:
+```
+ln -s $PWD '/filepath'
+```
+## Installing QGreenland Custom
+
+To install QGreenland Custom, navigate to plugins -> Installed -> Check box on QGreenland -> OK
+
+Instead of restarting QGIS after installing QGreenland Custom, 
```

Comment:
I think we should do more here to explain what plugin reloader does (enables developers to see their changes immediately by clicking "reload" instead of restarting QGIS).